src/main/webapp/availability.js

```diff
@@ -14,4 +14,27 @@
 
 function onAvailabilityLoad() {
   supplyLogoutLink();
+  loadAvailabilityTable(availabilityTableDiv(), browserTimezoneOffset());
+}
+
+// Toggles a tile from green to white and vice versa when clicked.
+function toggleTile(tile) {
+  tile.classList.toggle('table-success');
+}
+
+function loadAvailabilityTable(tableDiv, timezoneOffset) {
+  fetch('/availabilityTable.jsp?timeZoneOffset=' + timezoneOffset)
```

Comment:
Nit: Could be written with string interpolation `/availabilityTable.jsp?timeZoneOffset=${timezoneOffset}`

Reply:
Done